paymentsheet/src/test/java/com/stripe/android/link/LinkButtonScreenshotTest.kt

```diff
@@ -0,0 +1,35 @@
+package com.stripe.android.link
+
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import com.stripe.android.link.ui.LinkButton
+import com.stripe.android.utils.screenshots.PaparazziRule
+import com.stripe.android.utils.screenshots.SystemAppearance
+import org.junit.Rule
+import org.junit.Test
+
+internal class LinkButtonScreenshotTest {
+    @get:Rule
+    val paparazziRule = PaparazziRule(
+        arrayOf(SystemAppearance.LightTheme),
+        boxModifier = Modifier
+            .padding(0.dp)
```

Comment:
Based on the screenshots, the button has some internal padding. Can we get rid of that?

Reply:
This is done ✅